EIPS/eip-2544.md

```diff
@@ -3,7 +3,7 @@ eip: 2544
 title: ENS Wildcard Resolution
```

Comment:
```suggestion
title: ENS Wildcard Resolution
description: Extends ENS client behavior to support "wildcard" resolution of subdomains. This is accomplished by using a parent domain's resolver if none is set on a given subdomain.
```
We have removed the simple summary in favor of a `description` frontmatter field.

Reply:
Note: Please remove the `## Simple Summary` section as well (can't provide PR feedback on it since it wasn't touched in this PR).